jekyll/_cci2/non-aws.md

```diff
@@ -86,49 +86,49 @@ The following sections describe the steps for installation of the Services VM an
 
 ### Prerequisites
 
-Have the following information available before beginning the installation procedure:
+Have the following information/resources available before beginning the installation procedure:
 
-- CircleCI License file (.rli), contact CircleCI support for a license.
-- A machine to run Ubuntu 14.04 with a minimum of at least 100 GB storage, 32 GB RAM, and 4 CPUs (8 CPUs preferred) for the Services VM.
-- A cluster of machines running Ubuntu 14.04 with minumum of 8 GB RAM, 4 CPUs each, and network access any Docker registries that are required by your builds for the Nomad Client VMs.
+- CircleCI License file (.rli)—contact CircleCI support for a license
+- A machine to run Ubuntu 14.04 with a minimum of at least 100 GB storage, 32 GB RAM, and 4 CPUs (8 CPUs preferred) for the Services VM
+- A cluster of machines running Ubuntu 14.04 with a minumum of 8 GB RAM and 4 CPUs each, as well as network access to any Docker registries that are required by your builds for the Nomad Client VMs
 
 ### Steps
 
-To install the Services machine, perform the following steps:
+**To install the Services machine, perform the following steps:**
```

Comment:
We don't bold entire sentences. Let's add a heading instead
### Installing the Services Machine